src/compiler/checker.ts

```diff
@@ -41313,6 +41326,21 @@ namespace ts {
             return false;
         }
 
+        function checkGrammarImportCallArguments(node: ImportCall, nodeArguments: NodeArray<Expression>): boolean {
+            if (moduleKind !== ModuleKind.ESNext) {
+                if (nodeArguments.length !== 1) {
+                    return grammarErrorOnNode(node, Diagnostics.The_second_argument_of_an_import_call_is_only_supported_when_the_module_flag_is_not_available_in_the_current_module_flag);
+                }
+                checkGrammarForDisallowedTrailingComma(nodeArguments);

```

Comment:
We disallowed a trailing comma previously because it wasn't permitted in the specification text for any edition. We should do like we do for trailing commas in other calls now that it is permitted in the spec, but elide the trailing comma when we emit an import call.

Reply:
I'm not sure what happend but there's no errors if target es2015 and esnext.

Reply:
Oh. My bad.